vendor/github.com/kubernetes-incubator/service-catalog/cmd/svcat/broker/deregister_cmd.go

```diff
@@ -46,7 +46,7 @@ func NewDeregisterCmd(cxt *command.Context) *cobra.Command {
 	return cmd
 }
 
-// Validate checks that the required arguements have been provided
+// Validate checks that the required arguments have been provided
```

Comment:
We should make changes in vendored source code. This needs to be fixed upstream.

Reply:
@kadel We **shouldn't** right?

Reply:
> @kadel We **shouldn't** right?

ah, yes, we should NOT 😊